yarn.lock

```diff
@@ -108,6 +108,15 @@
     jsesc "^2.5.1"
     source-map "^0.5.0"
 
+"@babel/generator@^7.11.5":
```

Comment:
Some of these lockfile updates seem out-of-place :thinking: I can't see why this would be added, for example. Leftover from the `@babel/eslint-parser` install from earlier perhaps?

Reply:
I have just pushed a commit to get rid of these extraneous changes (eae46dca4de8fd97a32e5270c58654a2b3e7d9a0)

The steps I took were:
* Checkout the lockfile on develop (`git checkout develop yarn.lock`)
* Update `rtlcss` to v6.2.1 in the lockfile
* Run `yarn` to install `lavamoat-core` again